editor/editor_properties_array_dict.cpp

```diff
@@ -388,6 +393,15 @@ void EditorPropertyArray::update_property() {
 #endif
 }
 
+void EditorPropertyArray::_remove_pressed(int index) {
+	Variant array = object->get_array();
+	array.call("remove", index);
```

Comment:
Is there a specific reason to use `call` here instead of just `array.remove(index)` and `array.duplicate()`?
Also, does it really need to be duplicated?

Lastly, please name the parameter `p_index`.

Reply:
For the usage of call, I was following the pattern throughout other methods. See line 440, 438, 330 (331 then calls duplicate for no reason again, odd), 314...etc Maybe these are wrong.

The duplication occurs for undo\redo capability again copying the same pattern found elsewhere (Line 452).

Let me know the preferred method, I guess the other option is using Variant duplicate and\or operator Array() for remove(p_index)\duplicate. The main odd difference between get_array and get_dictionary is that get_array still only returns a Variant.

Reply:
Indeed, looks like the original code added by @reduz in eded061668 used `call()` and `duplicate()` indeed. I'm not sure exactly why it's needed, but I guess @reduz had a good reason for that, and you're correct to try to follow the patterns from the rest of the class.